build/integrationtests/Dockerfile

```diff
@@ -7,15 +7,21 @@ LABEL maintainer Rob Bovill <rbovill@lsst.org>
 WORKDIR ${HOME}
 
 ARG integrationtests
+ARG robot_efd
 
 RUN source /home/saluser/.setup_sal_env.sh && \
 mamba install -c lsstts ts-integrationtests=${integrationtests} -y && \
 mamba install -c conda-forge robotframework && \
-pip3 install -U lsst-efd-client
+pip3 install -U PyYAML lsst-efd-client
```

Comment:
are you sure `PyYAML` is not available in conda? I know it is small detail but it is better to try installing everything we can from conda.